controllers/ctltools.js

```diff
@@ -22,7 +22,10 @@ const trimWhitespaces = (object) => {
 
 const sanitizeToolInputs = (id, body) => {
     body.id = id;
+    body.description = body.description || undefined;
     body.url = body.url || undefined;
+    body.medium.mediumId = body.medium.mediumId || undefined;
```

Comment:
I just realized that mediumId is required, when medium is defined, so this might not work, when no medium is defined.